src/containers/PrivacyPolicy/index.tsx

```diff
@@ -0,0 +1,341 @@
+          We will collect any information contained in such feedback and will treat the personal data in it in
+          accordance with this Policy.
+        </p>
+        <h2 className="PrivacyPolicy__subtitle">International Data Transfer</h2>
+        <p>
+          Your information, including personal data that we collect from you, may be transferred to, stored at and
+          processed by us outside the country in which you reside, where data protection and privacy regulations may not
+          offer the same level of protection as in other parts of the world. By accepting this Policy, you agree to this
+          transfer, storing or processing. We will take all steps reasonably necessary to ensure that your data is
+          treated securely and in accordance with this Policy.
+        </p>
+        <h2 className="PrivacyPolicy__subtitle">Security</h2>
+        <p>
+          We seek to use reasonable organizational, technical and administrative measures to protect personal data
+          within our organization. Unfortunately, no transmission or storage system can be guaranteed to be completely
+          secure, and transmission of information via the Internet is not completely secure. If you have reason to
+          believe that your interaction with us is no longer secure (for example, if you feel that the security of any
+          account you might have with us has been compromised), please immediately notify us of the problem by
+          contacting us.
+        </p>
+        <h2 className="PrivacyPolicy__subtitle">Retention</h2>
+        <p>
+          We will only retain your personal data as long reasonably required for you to use the website until you close
+          your account/cancel your subscription unless a longer retention period is required or permitted by law (for
+          example for regulatory purposes).
+        </p>
+        <h2 className="PrivacyPolicy__subtitle">Our Policy on Children</h2>
+        <div>
+          Our website is/are not directed to children under 16. If a parent or guardian becomes aware that his or her
+          child has provided us with information without their consent, he or she should contact us. We will delete such
+          information from our files as soon as reasonably practicable.
+          <br />
+          <br />
+          <strong>Your Rights</strong>
+          <ul className="PrivacyPolicy__list">
+            <li>
+              <strong>Opt-out.</strong> You may contact us anytime to opt-out of: (i) direct marketing communications;
+              (ii) automated decision-making and/or profiling; (iii) our collection of sensitive personal data; (iv) any
+              new processing of your personal data that we may carry out beyond the original purpose; or (v) the
+              transfer of your personal data outside the EEA. Please note that your use of some of the website may be
```

Comment:
EEA? Not sure this statement is in accordance to GDPR if the purpose is that one. I'd suggest a review on that.
some => some parts